lib/Backend/LowerMDShared.cpp

```diff
@@ -1512,22 +1521,25 @@ LowererMD::Legalize(IR::Instr *const instr, bool fPostRegAlloc)
             break;
 
         case Js::OpCode::ADDSD:
-        case Js::OpCode::ADDPS:
         case Js::OpCode::ADDPD:
         case Js::OpCode::SUBSD:
+        case Js::OpCode::ANDPD:
+        case Js::OpCode::ANDNPD:
+        case Js::OpCode::DIVPD:
+        case Js::OpCode::MAXPD:
+        case Js::OpCode::MINPD:
+        case Js::OpCode::MULPD:
+        case Js::OpCode::SUBPD:
+            Assert(AutoSystemInfo::Data.SSE2Available());
```

Comment:
Maybe these asserts should be in DbCheckLower?  We should add a machine dependent for it...